cmd/server/main.go

```diff
@@ -88,6 +88,9 @@ func main() {
 			deliveryServices = append(deliveryServices, fcm)
 		}
 
+		expo := delivery.NewExpoDelivery(logger, opts.Expo)
```

Comment:
To get this merged in, we'll need it to be conditional on opts.Expo.Enabled being set. Otherwise it will cause errors with non-expo users.